miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSampleService.java

```diff
@@ -381,7 +381,6 @@ private DetailedSample findOrCreateParent(DetailedSample sample) throws IOExcept
   private SampleTissue createParentTissue(SampleTissue tissue, DetailedSample child) throws IOException {
     log.debug("Creating a new Tissue to use as a parent.");
     tissue.setProject(child.getProject());
-    tissue.setDescription("Tissue");
```

Comment:
That's what the 'isSynthetic' field is for, and a big grey box explaining it shows up in the UI. The ghosts' old descriptions didn't mention that they were ghosts anyway - just the sample category, which again is available elsewhere